### DISCUSSION TITLE
文件不能创建、配置不能读取、日志打不开、除了收能量其他功能正常，请看这里

### DISCUSSION THREAD

**Discussion by constanline** [constanline/XQuickEnergy#282] [General/我有一言当讲否]
文件不能创建、配置不能读取、日志打不开、除了收能量其他功能正常，目前看到的反馈均属于权限问题。
1、单开的话请确认《支付宝》和《芝麻粒》的”始终允许读写手机存储“权限是否给到。
2、多开的话除了确认权限，请尽量使用手机自带分身。如果没有自带分身，请尝试换用其他的分身软件，目前发现部分使用者在用LSP自带的多开会出现没有权限的问题。
3、如果你没法确认自己的实际权限情况，可以尝试由支付宝创建config文件(即删掉config文件，然后别开芝麻粒，先开支付宝)，其他文件(如forest、farm、statistics等)如果出现支付宝没有日志的情况，同样可以尝试这个办法。

**Comment by JiangAshao** [comment]
多开的话 重新安装后，会出现假授权。操作 取消授权-再次授权就可以获取到权限了
